src/server/cli/vdiff/migrate.js

```diff
@@ -30,7 +30,9 @@ async function start(argv = []) {
 				.replace(/^d2l-/, '')
 				.replace(new RegExp(`^${dirName}-`), '');
 
-			const newDir = dir.replace(`${oldSuffix}/${dirName}`, `${PATHS.GOLDEN}/${dirName}/chromium`);
+			const newDir = local ?
+				`${PATHS.VDIFF_ROOT}/${dir.replace(`${oldSuffix}/${dirName}`, `${dirName}/${PATHS.GOLDEN}/chromium`)}` :
+				dir.replace(`${oldSuffix}/${dirName}`, `${PATHS.GOLDEN}/${dirName}/chromium`);
 
 			await mkdir(newDir, { recursive: true });
 			return rename(file, join(newDir, newName));
```

Comment:
I think the migration is working, but then line 40 below deletes the whole test folder that `screenshots` was in?